openpype/hosts/houdini/api/plugin.py

```diff
@@ -222,25 +223,44 @@ def collect_instances(self):
         self.cache_subsets(self.collection_shared_data)
         for instance in self.collection_shared_data[
                 "houdini_cached_subsets"].get(self.identifier, []):
+
+            node_data = read(instance)
+
+            # Node paths are always the full node path since that is unique
+            # Because it's the node's path it's not written into attributes
+            # but explicitly collected
+            node_path = instance.path()
+            node_data["instance_id"] = node_path
+            node_data["instance_node"] = node_path
+
             created_instance = CreatedInstance.from_existing(
-                read(instance), self
+                node_data, self
             )
             self._add_instance_to_context(created_instance)
 
     def update_instances(self, update_list):
         for created_inst, changes in update_list:
             instance_node = hou.node(created_inst.get("instance_node"))
-
             new_values = {
                 key: changes[key].new_value
                 for key in changes.changed_keys
             }
-            imprint(
+            self.imprint(
                 instance_node,
                 new_values,
                 update=True
             )
 
+    def imprint(self, node, values, update=False):
+        # Update values to be dynamic
+
+        values["subset"] = '`opname(".")`'
+        values["variant"] = '`strreplace(chs("subset"), chs("family"), "")`'
```

Comment:
As explained [here](https://github.com/ynput/OpenPype/pull/5666#issuecomment-1741710742) this isn't a correct approach unfortunately.